### PR TITLE
Document init

### PR DESCRIPTION
**Pull request by joakim-hove**
+ Write NTG. Completely untested.

### CODE REVIEW

**Comment by andlaus** [comment]
I guess this also needs some simulator side support to be useful?

**Comment by joakim-hove** [comment]
Well. - if the simulator wants to output something funky the simulator must calculate that funky property and pass it in the `simProps` object. But API wise this is ready and has been for quite some time.

Of course I have to fix the compile issue, and there will be updated to the reference data (I think) - but then this can be merged.

**Comment by andlaus** [comment]
> this is ready and has been for quite some time.

Does this mean that NTG will be written by flow* after this PR?

**Comment by joakim-hove** [comment]
Well - iff NTG is part of the deck: yes.

15. feb. 2017 18:46 skrev "Andreas Lauser" <notifications@github.com>:

**Comment by andlaus** [comment]
cool :+1:

File: opm/output/eclipse/EclipseIO.hpp [hunk 1]
```diff
@@ -53,19 +53,53 @@ class EclipseIO {
...
-    ///   calculated by the simulator, e.g. the transmissibility
-    ///   vectors TRANX, TRANY and TRANZ.
-    ///
-    /// - The NNC argument is distributed between the EGRID and INIT
-    ///   files.
-    ///
-    /// If you want FOE in the SUMMARY section, you must pass the initial
-    /// oil-in-place "OIP" key in Solution
-    void writeInitial( data::Solution = data::Solution(), const NNC& = NNC());
+    /*
```
Comment (atgeirr) [inline opm/output/eclipse/EclipseIO.hpp]: Just a minor thing: this changes the doc string from being a doxygen doc string to just a regular comment. Was that intended?

Comment (joakim-hove) [reply opm/output/eclipse/EclipseIO.hpp]: Not really - I'll revert that part of the commit.

**Comment by alfbr** [comment]
That was fast, thanks!

>  iff NTG is part of the deck: yes.

Actually, I believe NTG is written out regardless. I can double check tomorrow and report back. According to the documentation, it is defaulted to 1 for every cell after the GRID section.

**Comment by alfbr** [comment]
I can confirm that NTG is always written out to the INIT file. If it is not set in the input files, then all values are defaulted to 1.

**Comment by joakim-hove** [comment]
> I can confirm that NTG is always written out to the INIT file. If it is not set in the input files, then all values are defaulted to 1.

OK good - it should now be written unconditionally to the INIT file.

**Comment by alfbr** [comment]
Great, then I have no further comments.

**Comment by joakim-hove** [comment]
Some fiddling in parser required first: https://github.com/OPM/opm-parser/pull/1043

**Comment by joakim-hove** [comment]
jenkins build this with downstreams opm-parser=1043 please

**Review by pgdr — APPROVED** [review]
Looks good!  Merge!

**Pull request merged**